src/Faction/ui/FactionRoot.tsx

```diff
@@ -121,9 +125,23 @@ function MainPage({ faction, rerender, onAugmentations }: IMainProps): React.Rea
         {faction.name}
       </Typography>
       <Info faction={faction} factionInfo={factionInfo} />
-      {canAccessGang && (
+      {isManageGangVisible && (
         <>
-          <Option buttonText={"Manage Gang"} infoText={gangInfo} onClick={manageGang} />
+          <Box>
+            <Paper sx={{ my: 1, p: 1 }}>
+              <Tooltip
+                title={!isManageGangClickable ? <Typography>Unlocked when reaching -54000 karma</Typography> : ""}
```

Comment:
do we have a constant to back this on? `-54000` just incase the number was changed someday in the future

Reply:
Not at that point, it's currently a const in PlayerObjectGangMethod.ts. Declared right before canAccessGang function's definition.

Reply:
I pushed a commit to streamline that issue.